coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 99.42
+    "covered_percent": 74.9
```

Comment:
@tmikeschu  When running the full rspec suite, and viewing the simplecov index in the browser, we get an overall test coverage of 99%. Not sure what this line is doing, but it is not indicative of our testing coverage.